jenkins/scripts/select-compiler.sh

```diff
@@ -8,13 +8,16 @@ if [ "$DONTSELECT_COMPILER" != "DONT" ]; then
   esac
 fi
 
+# get node version
+if [ "${NODE_MAJOR_VERSION}X" = "X" ]; then
```

Comment:
Since we know this is going to run in bash, [SO has a very high upvote](https://stackoverflow.com/a/13864829/27955) for using
```bash
if [ -z ${NODE_MAJOR_VERSION+x} ]; then
```